AlicerceTests/ServiceLocatorTests.swift

```diff
@@ -0,0 +1,301 @@
+//
+//  ServiceLocatorTests.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 26/01/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+class ServiceLocatorTests : XCTestCase {
+    
+    var serviceLocator = ServiceLocator()
+    
+    override func setUp() {
+        super.setUp()
+        
+        serviceLocator = ServiceLocator()
+    }
+    
+    func testRegisterUsingServiceTypeInference() {
```

Comment:
yes, makes a lot of sense. that is the problem when we use testing frameworks 😄 

👍